eng/common/templates/steps/source-build.yml

```diff
@@ -34,9 +34,14 @@ steps:
       targetRidArgs='/p:TargetRid=${{ parameters.platform.targetRID }}'
     fi
 
+    publishArgs=
```

Comment:
@JunTaoLuo - I will migrate these changes to Arcade but would like to validate them with these changes first.

Reply:
I also reverted your changes to the build.sh script to support the '--publish' option.

Reply:
The changes to common are being added with https://github.com/dotnet/arcade/pull/7427

Reply:
Suggest reverting changes to eng/common/ files in this PR and instead triggering our Arcade subscription. @MichaelSimons change went in a couple of days ago.

Reply:
Updated